.github/actions/generate-coverage/tests/test_scripts.py

```
@@ -0,0 +1,87 @@
+        "DETECTED_LANG": "rust",
+        "DETECTED_FMT": "lcov",
+        "GITHUB_OUTPUT": str(tmp_path / "gh.txt"),
+    }
+    script = Path(__file__).resolve().parents[1] / "scripts" / "run_rust.py"
+    res = run_script(script, env)
+    assert res.returncode == 2
+    assert "cargo llvm-cov failed" in res.stderr
+
+
+
+def test_merge_cobertura(tmp_path, shell_stubs):
```

Ruff (0.11.9)

Missing return type annotation for public function `test_merge_cobertura`

Add return type annotation: `None`

(ANN201)